src/pypeh/adapters/outbound/export/xlsx.py

```diff
@@ -33,7 +33,7 @@
 }
 
 
-def get_observable_property_property(varname, observable_property, codebook_property_name):
+def get_observable_property_property(label, observable_property, codebook_property_name):
```

Comment:
I assume this is required for the update to the new version of the PEH data model. In the future, it might be best to do this in a separate PR.